packages/editor-ui/src/views/NodeView.vue

```diff
@@ -1682,7 +1678,8 @@ export default mixins(
 							// If adding more than one node, offset the X position
 							mousePosition[0] -
 								NodeViewUtils.NODE_SIZE / 2 +
-								NodeViewUtils.NODE_SIZE * (index * 2 + NodeViewUtils.GRID_SIZE),
+								NodeViewUtils.NODE_SIZE * index * 2 +
+								NodeViewUtils.GRID_SIZE,
```

Comment:
was this intentional change?

Reply:
Yes. It fixes the node insert position when dropping